src/commands/inventory/push/readme.ts

```diff
@@ -52,7 +56,9 @@ const examples = buildExamples<PushCommandFlags>(
 
 export default `#### Scopes
 
-The scopes for \`transcend inventory push\` are the same as the scopes for [\`transcend inventory pull\`](#transcend-inventory-pull).
+The API key permissions for this command vary based on the \`resources\` argument:
```

Comment:
re this [comment](https://github.com/transcend-io/cli/pull/446/files#r2277952600)

```suggestion
The API key permissions for this command vary based on the resources declared as top-level keys in your [`transcend.yml`](#transcendyml) file:
```